Api/ATM/ATM2/Request.php

```diff
@@ -0,0 +1,279 @@
+<?php
+
+namespace baibaratsky\WebMoney\Api\ATM\ATM2;
+
+use baibaratsky\WebMoney\Api\ATM;
+use baibaratsky\WebMoney\Exception\ApiException;
+use baibaratsky\WebMoney\Request\RequestValidator;
+use baibaratsky\WebMoney\Signer;
+
+/**
+ * Class Request
+ *
+ * @link http://wiki.wmtransfer.com/projects/webmoney/wiki/Interface_ATM2
+ */
+class Request extends ATM\Request
+{
+    const TRANSACTION_TYPE_REAL = 0;
+    const TRANSACTION_TYPE_TEST = 1;
+
+    /** @var int payment/@id */
+    protected $transactionId;
+
+    /** @var string payment/@currency */
+    protected $currency;
+
+    /** @var int payment/@test */
+    protected $test;
+
+    /** @var string payment/@exchange */
+    protected $exchange;
+
+    /** @var string payment/purse */
+    protected $payeePurse;
+
+    /** @var int payment/phone */
+    protected $phone = 0;
+
+    /** @var float payment/price */
+    protected $price;
+
+    /** @var string payment/date */
+    protected $date;
+
+    /** @var int payment/point */
+    protected $point;
+
+    /**
+     * @param string $authType
+     *
+     * @throws ApiException
+     */
+    public function __construct($authType = self::AUTH_CLASSIC)
+    {
+        switch ($authType) {
+            case self::AUTH_CLASSIC:
+                $this->url = 'https://transfer.gdcert.com/ATM/Xml/Payment2.ashx';
+                break;
+
+            case self::AUTH_LIGHT:
+                $this->url = 'https://transfer.gdcert.com/ATM/Xml/Payment2.ashx';
+                $this->signature = base64_encode(
```

Comment:
This is not right way to initialise signature, I should move it to cert() method.